### PR TITLE
confluent-platform:5.4.1 Added libexec to installation

### PR DESCRIPTION
**Pull request by nick-manasys**
Added the following to .bashrc to make "confluent" script work

export CONFLUENT_HOME="/usr/local/Cellar/confluent-platform/5.4.1/"
export PATH="$PATH:$CONFLUENT_HOME/bin"

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Does it work with a higher java version already?

**Comment by nick-manasys** [comment]
> Does it work with a higher java version already?

No. I have only tested it with 1.8

Since package was renamed, confluent-oss -> confluent-platform, I have not had a working install via hombrew. It seems to me that the "libexec" folder is missing from the install. By adding that folder and setting CONFLUENT_HOME and adding the bin directory I got it to work.

**Comment by SMillerDev** [comment]
Can you check if it works with a higher Java version?

**Comment by nick-manasys** [comment]
Is that not a new ticket? I am just fixing the current version to work with 1.8.

[many minutes later]
I tested with other more recent versions of java. It works with 1.8 and 11 as suggested by the error message for the other versions:

ERROR: The Confluent CLI requires Java version 1.8 or 1.11.

1.8 and 11 work. I tested manually by running "confluent local start/status/stop"

I have not updated the dependency which is on 1.8

**Comment by SMillerDev** [comment]
> Is that not a new ticket? I am just fixing the current version to work with 1.8.

As a maintainer I always look through the entire formula to find lingering issues whenever someone fixes something.

**Comment by SMillerDev** [comment]
Does the current test block pass without your fix? If so, is there a better way to test if the tool is working?

**Comment by nick-manasys** [comment]
It does pass as it is not really testing much. The deployment cannot work without the directory that I added as it misses some OS dependent scripts. The "confluent" script in "/bin" invokes /libexec/cli/darwin_amd64/confluent when used on MacOS for example.

**Comment by SMillerDev** [comment]
Is there a way to run `confluent local start`, calling the service and stopping it again in the test? To make sure this doesn't break again next time?

**Comment by nick-manasys** [comment]
I am trying to start confluent but it cannot create its configuration directory ~/.confluent
How do I create this directory (/Users/brew/.confluent)?

File: Formula/confluent-platform.rb [hunk 1]
```diff
@@ -15,10 +15,22 @@ def install
...
     prefix.install "share"
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/kafka-broker-api-versions --version")
+
+    home = ENV["HOME"]
+    system "echo", "#{home}/.confluent"
+    system "mkdir", "#{home}/.confluent/"
+    system "ls", "#{home}/.confluent"
```
Comment (SMillerDev) [inline Formula/confluent-platform.rb:28]:
```suggestion
    system "mkdir", "#{ENV["HOME"]}/.confluent/"
```

File: Formula/confluent-platform.rb [hunk 2]
```diff
@@ -15,10 +15,22 @@ def install
...
+
+    ENV["CONFLUENT_HOME"] = "/usr/local/Cellar/confluent-platform/5.4.1/"
+    system "#{bin}/confluent", "--version"
```
Comment (SMillerDev) [inline Formula/confluent-platform.rb:31]: Didn't we have this before and it failed to catch the error?

File: Formula/confluent-platform.rb [hunk 3]
```diff
@@ -15,10 +15,22 @@ def install
...
+    assert_equal "0\n", shell_output("echo $?")
+    home = ENV["HOME"]
+    system "echo", "#{home}/.confluent"
```
Comment (SMillerDev) [inline Formula/confluent-platform.rb:34]: This isn't needed

**Comment by nick-manasys** [comment]
Running the second test "confluent status" gives an error

"mkdir /Users/brew/.confluent: operation not permitted"

The build server user cannot create a directory in its home directory for the configuration. I cannot fix this.

File: Formula/confluent-platform.rb (hunk 3 above)
Comment (nick-manasys) [reply Formula/confluent-platform.rb]: I was trying to get the build to work. Please read my comment below.

My change makes the install work. I have run and tested it on several machines. However, the build server fails the test due to a permission problem creating the "~/.confluent". This is due to the build environment. There was no test to start it using confluent.

**Pull request closed without merging**